WooCommerce/Classes/POS/Presentation/Reusable Views/WavesProgressViewStyle.swift

```diff
@@ -0,0 +1,96 @@
+import SwiftUI
+
+struct WaveProgressViewStyle: ProgressViewStyle {
+    func makeBody(configuration: Configuration) -> some View {
+        WaveProgressView(configuration: configuration)
+    }
+}
+
+private struct WaveProgressView: View {
+    let configuration: ProgressViewStyleConfiguration
+    private let animationDuration: Double = 0.5
+    private let radii: [CGFloat] = [40.0, 70.0, 100.0]
+    private let inactiveInset: CGFloat = 0.025
+
+    @State private var activeArcIndex: Int = 0
+
+    private var arcCount: Int {
+        radii.count
+    }
+
+    private var xOffset: CGFloat {
+        guard let lastRadius = radii.last,
+              let firstRadius = radii.first,
+              lastRadius > firstRadius else {
+            return 0
+        }
+        let centreOfWaves = (lastRadius - firstRadius) / 2
+        return -((firstRadius + centreOfWaves) / 2)
+    }
+
+    var body: some View {
+        ZStack {
+            ForEach(0..<arcCount, id: \.self) { index in
+                let isActive = index == activeArcIndex
+                let radius = radii[index]
+
+                ArcShape(startAngle: .degrees(333),
+                         endAngle: .degrees(27))
+                .trim(from: isActive ? 0.0 : 0 + inactiveInset,
+                      to: isActive ? 1.0 : 1 - inactiveInset)
+                .stroke(isActive ? Color(.wooCommercePurple(.shade60)) : Color(.wooCommercePurple(.shade40)),
```

Comment:
I think we can use static colors which do not change in dark mode. Especially given the previous tap/swipe state uses a similar icon which remains static in dark mode.

```suggestion
                .stroke(isActive ? Color(.withColorStudio(.wooCommercePurple, shade: .shade60)) : Color(.wooCommercePurple(.shade40)),
```

Reply:
👍

Reply:
Done in c2c3906